src/db.ts

```diff
@@ -56,43 +57,41 @@ class DbHandler {
         const roomModel = getModelForClass(RoomSchema);  
         try {
             const res = await roomModel.updateOne({ _id: roomId }, updateData);
-            console.log(`Room updated with id (handler): ${roomId}`);
+            console.log(`Room successfully updated with id: ${roomId}, collection: ${roomModel.collection.collectionName}, updated fields: ${Object.keys(updateData).join(', ')}`);
         } catch (error) {
-            console.error("Room update error (handler):", error);
+            console.error(`Failed to update room fields: ${Object.keys(updateData).join(', ')}, with id: ${roomId}, collection: ${roomModel.collection.collectionName}, error:`, error);
         }
     }
 
-    public async createQuiz(quizData: QuizSchema): Promise<void> {
+    public async createQuizQuestionIds(quizData: QuizSchema): Promise<void> {
         try {
             const quizModel = getModelForClass(QuizSchema);
             const quiz = await quizModel.create(quizData);
-            await quiz.save();
-            console.log(`New quiz created with id (handler): ${quiz._id}`);
+            console.log(`New quiz created with id: ${quiz._id}, collection: ${quizModel.collection.collectionName}`);
         } catch (error) {
-            console.error("Quiz create error (handler):", error);
+            console.error(`Quiz creation error:`, error);
         }
     }
 
-    public async updateQuizList(quizId: string, updateData: {
+    public async updateQuizQuestionIds(quizId: string, updateData: {
         Questions?: Array<string>,
     }): Promise<void> {
         const quizModel = getModelForClass(QuizSchema);
         try {
             const res = await quizModel.updateOne({ _id: quizId }, updateData);
-            console.log(`Quiz updated with id (handler): ${quizId}`);
+            console.log(`Quiz successfully updated with id: ${quizId}, collection: ${quizModel.collection.collectionName}, updated fields: ${Object.keys(updateData).join(', ')}`);
         } catch (error) {
-            console.error("Quiz update error (handler):", error);
+            console.error(`Failed to update quiz fields: ${Object.keys(updateData).join(', ')}, with id: ${quizId}, collection: ${quizModel.collection.collectionName}, error:`, error);
         }
     }
 
     public async createQuestionHandler(questionData: QuestionSchema): Promise<void> {
```

Comment:
Please rename this function to `createQuestion`, I don't understand why its called `createQuestionHandler`